src/auth/okta_helper.py

```diff
@@ -0,0 +1,121 @@
+from app import *
+from configparser import ConfigParser
+from okta_jwt_verifier import JWTVerifier
+import asyncio
+import os
+
+loop = asyncio.get_event_loop()
+
+
+class OktaHelper:
+
+    """ Okta token authorization helper. 
+
+    The okta token authorization helper will use okta jwt verifier
+    library to verify whether an access token or id token is valid.
+
+    Attributes: 
+        okta_configs: A dictionary of okta configs from `config/setup.cfg`.
+                      The key is a string field name such as `auth_uri`, 
+                      and the value is corresponding config value.
+    """
+
+    def __init__(self):
+        """ Initialize okta config.
+
+        Initialize okta config parsing project config file and fetch okta config.
+        """
+
+        configs = ConfigParser()
+        configs.read(os.path.dirname(os.path.abspath(__file__))
+                     + '/../../config/setup.cfg')
+        self.okta_configs = configs['okta']
+
+    def is_access_token_valid(self, token):
+        """ Check whether an access token is valid.
+
+        Args:
+            token: str, okta access token
+
+        Returns:
+            bool, whether the okta access token is valid.
+            str, the user id if exists. Otherwise, return None.
+        """
+
+        jwt_verifier = JWTVerifier(self.okta_configs['issuer'],
+                                   self.okta_configs['client_id'],
+                                   'api://default')
+        try:
+            loop.run_until_complete(jwt_verifier.verify_access_token(token))
```

Comment:
I tried to generate an access token using postman and put the authorization token into the header (followed steps [here](https://developer.okta.com/docs/guides/implement-oauth-for-okta/request-access-token/)). It throws an exception 'No matching JWK' at line 49, did I miss any steps?

Reply:
In postman,
URL: `http://127.0.0.1:5000/example_access_token_check` or `http://127.0.0.1:5000/example_id_token_check`
Click `Authorization`, type `Bearer Token`, and then put your token in the token field. Now send the request, and you should not expect an exception.